packages/datadog_flutter_plugin/lib/src/datadog_noop_platform.dart

```diff
@@ -0,0 +1,60 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-Present Datadog, Inc.
+
+import '../datadog_internal.dart';
+import 'datadog_configuration.dart';
+
+class DatadogSdkNoopPlatform extends DatadogSdkPlatform {
```

Comment:
tiny change for the naming consistency

```suggestion
class DatadogSdkNoOpPlatform extends DatadogSdkPlatform {
```